tidy3d/plugins/smatrix/component_modelers/terminal.py

```diff
@@ -55,13 +70,18 @@ class TerminalComponentModeler(AbstractComponentModeler[NetworkIndex, NetworkEle
     assume_ideal_excitation: bool = pd.Field(
         False,
         title="Assume Ideal Excitation",
-        description="If ``True``, only the excited port is assumed to have incident power, so the "
-        "vector of incident power wave amplitudes (a) is assumed to be all zeros except for the "
-        "entry associated with the excited port. This choice simplifies the calculation of the "
-        "scattering matrix. If ``False``, every entry in the vector of incident power wave "
-        "amplitudes (a) is calculated explicitly. This choice requires a matrix inversion when "
-        "calculating the scattering matrix, but may lead to more accurate scattering parameters "
-        "when there are reflections from simulation boundaries. ",
+        description="If ``True``, only the excited port is assumed to have a nonzero incident wave "
+        "amplitude power. This choice simplifies the calculation of the scattering matrix. "
+        "If ``False``, every entry in the vector of incident wave amplitudes (a) is calculated "
+        "explicitly. This choice requires a matrix inversion when calculating the scattering "
+        "matrix, but may lead to more accurate scattering parameters when there are "
+        "reflections from simulation boundaries. ",
+    )
+
+    S_def: SDef = pd.Field(
```

Comment:
trying to compare the following terminologies for the field name: `wave_choice`, `basis_choice`, `s_param_def`, and the current one.